model/manifest.go

```diff
@@ -13,13 +13,41 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	PLUGIN_CONFIG_TYPE_TEXT      = "text"
+	PLUGIN_CONFIG_TYPE_BOOL      = "bool"
+	PLUGIN_CONFIG_TYPE_RADIO     = "radio"
+	PLUGIN_CONFIG_TYPE_DROPDOWN  = "dropdown"
```

Comment:
Do we need a way to specify auto-completed fields? Such as the JIRA plugin's user field?

Reply:
We could but we can also just use a generic text field for that and have the admin type it in for now